### DISCUSSION TITLE
Does this works for BTP Neo

### DISCUSSION THREAD

**Discussion by kumarniti4** [SAP/terraform-provider-btp#370] [General]
There's a use case where we want to spin a subaccount in a different region in case of downtime.
This would come really handy, hence I want to know does terraform for BTP supports Neo Environment or not.

**Comment by rui1610** [comment]
It supports those environments, that are supported by the BTP CLI. 
As the BTP CLI doesn't support Neo, the Terraform provider for SAP BTP doesn't support Neo as well.
